tasks/ports/freetds.rb

```diff
@@ -23,7 +23,7 @@ def configure_defaults
       end
 
       if windows?
-        opts << '--sysconfdir=C:\Sites'
+        opts << '--sysconfdir=C:/Sites'
```

Comment:
Is this going to work? According to Msys docs, paths starting with C: are considered to be native windows paths and do not get the benefit of automatic path conversion.